### PR TITLE
Release 0.13.2

### PR DESCRIPTION
**Pull request by FlorianRappl**
# New Pull Request

For more information, see the `CONTRIBUTING` guide.

## Prerequisites

Please make sure you can check the following boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project
- [x] All new and existing tests passed

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [x] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have updated the documentation accordingly
- [x] I have added tests to cover my changes

### Description

This is the second patch for 0.13. The major achievement here is that Piral.Blazor finally reached a ready status, which means that it can be fully used. In existing app shells or new app shells. Other than that only a few bug changes and cosmetic enhancements have been introduced.

### Remarks

Piral.Blazor will be released independently.

### CODE REVIEW

**Review by manuelroemer — APPROVED** [review]
LGTM! 🚀

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Piral Changelog
 
+## 0.13.2 (tbd)
```
Comment (nmalyschkin) [inline CHANGELOG.md:3]: enter date here

**Review by nmalyschkin — APPROVED** [review]

**Comment by nmalyschkin** [comment]
Looks great 👍

File: CHANGELOG.md (hunk 1 above)
Comment (FlorianRappl) [reply CHANGELOG.md:3]: This is filled out automatically.

Was always like that.

**Pull request merged**